.github/workflows/ci.yml

```diff
@@ -19,6 +19,7 @@ concurrency:
 env:
   CARGO_TERM_COLOR: always
   CARGO_INCREMENTAL: 0
+  CARGO_REGISTRIES_CRATES_IO_PROTOCOL: sparse
```

Comment:
We don't need this environment variable, as the `sparse` protocol has been the default since Rust v1.70.0:

https://blog.rust-lang.org/2023/06/01/Rust-1.70.0.html#sparse-by-default-for-cratesio

Reply:
reverted